README.md

```diff
@@ -162,33 +162,36 @@ The `has_ancestry` method supports the following options:
     :ancestry_column       Column name to store ancestry
                            'ancestry' (default)
     :ancestry_format       Format for ancestry column (see Ancestry Formats section):
-                           :materialized_path (default)     1/2/3, root nodes ancestry=nil
-                           :materialized_path2 (preferred) /1/2/3/, root nodes ancestry=/
+                           :materialized_path   1/2/3, root nodes ancestry=nil (default)
+                           :materialized_path2  /1/2/3/, root nodes ancestry=/ (preferred)
     :orphan_strategy       Instruct Ancestry what to do with children of a node that is destroyed:
                            :destroy   All children are destroyed as well (default)
                            :rootify   The children of the destroyed node become root nodes
                            :restrict  An AncestryException is raised if any children exist
                            :adopt     The orphan subtree is added to the parent of the deleted node
                                       If the deleted node is Root, then rootify the orphan subtree
+                           :none      skip this logic. (add your own `before_destroy`)
+                           String     Custom. `before_destroy :apply_orphan_strategy_#{String}` to handle orphans.
```

Comment:
This might need a little expansion so that people know they don't need to add their own `before_destroy` block here. Perhaps:

Your custom destroy strategy method named with the convention `apply_orphan_strategy_#{String}` will be executed as an `before_destroy` callback.

Reply:
Yea, After trying to document this had me wanting to drop the feature `apply_orphan_strategy_#{string}` and just have the person write a `before_destroy`

Reply:
Yea that makes sense :)